src/components/events/details/EventAccessList.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div class="row">
+    <div class="col">
+      <div class="card">
+        <TableHead :name="title" :links="tableHeadLinks" />
+        <TableBodyNoData v-if="items.length === 0" />
+        <div v-else class="list-group list-group-flush">
+          <div class="list-group-item" v-for="(item, index) in items" :key="index">
+            <div class="row">
+              <div class="col-2 align-self-center">
+                <router-link
+                  :to="{ name: 'EventAccessDetails', params: { id, userId: item.user.id } }"
+                >
+                  {{ item.user.name }}
+                </router-link>
+              </div>
+              <div
+                class="col align-self-center text-center"
+                v-for="permission in permissions"
+                :key="permission.id"
+              >
+                <i class="fas" :class="getPermissionIcon(item.permissions, permission.id)"></i>
+                <small class="ml-2">{{ permission.name }}</small>
+              </div>
+            </div>
+          </div>
+          <!-- <div class="list-group-item">
+            <div class="row">
+              <div class="col-2 align-self-center">
+                <a href="/events/1/users/2">User 1</a>
+              </div>
+              <div class="col align-self-center text-center">
+                <i class="fas fa-check text-success"></i>
+                <small class="ml-2">View Event</small>
+              </div>
+              <div class="col align-self-center text-center"></div>
+              <div class="col align-self-center text-center"></div>
+              <div class="col align-self-center text-center"></div>
+              <div class="col align-self-center text-center"></div>
+            </div>
+          </div> -->
```

Comment:
Remove this comment.

Reply:
Ok, this will be solved in the next commit.